### PR TITLE
Text improvements

### PR DESCRIPTION
**Pull request by rempack-b**
Improves the general appearance of text by rendering subpixels in shades of gray.

This required a fix to color::gray32. An off by one error made white render as 0xfffaff, which broke alpha rendering for text backgrounds that expected pure white.

### CODE REVIEW

File: src/rmkit/fb/stb_text.cpy [hunk 1]
```diff
@@ -141,7 +141,8 @@ namespace stbtext:
     for j = 0; j < image.h; j++:
       for i = 0; i < image.w; i++:
         uint32_t val = text_buffer[j*image.w+i]
-        image.buffer[j*image.w+i] = val == 0 ? WHITE: BLACK;
+        //image.buffer[j*image.w+i] = val == 0 ? WHITE: BLACK;
```
Comment (Eeems) [inline src/rmkit/fb/stb_text.cpy:144]: Remove instead of commenting out?

Comment (rempack-b) [reply src/rmkit/fb/stb_text.cpy:144]: I left it in case someone else wanted the old behavior back, but I'm fine to remove it

Comment (Eeems) [reply src/rmkit/fb/stb_text.cpy]: Isn't that usually that's what git history, using an older version of the library, or adding a feature flag is for?

**Comment by raisjn** [comment]
i tested this and its not a drop in replacement for current behavior, i think we will likely need to make this an optional way of rendering anti-aliased text

**Comment by raisjn** [comment]
added in 04728fea36fa7f8a392d7c50b89c8a852ec5a308, thanks!

**Pull request closed without merging**